lib/sengoku/tile.ex

```diff
@@ -48,4 +48,8 @@ defmodule Sengoku.Tile do
     |> Enum.into(%{})
     |> Map.keys
   end
+
+  def owned_by_player_id?(tile, player_id) do
+    tile.owner == player_id
+  end
```

Comment:
So the pattern-matching jiggery-pokery I mentioned was going to be something like this:

```elixir
def owned_by_player_id?(%{owner: player_id}, player_id), do: true
def owned_by_player_id?(_, _), do: false
```

But I think that's actually less clear than what you've got here. I just think it's cool that you can pattern match in two different arguments (or parts of arguments!) being the same thing.